model:

  algorithm: KNN
  max_num_faces: 1
